charts/prod-ohstaff-values.yaml

```diff
@@ -56,6 +56,7 @@ django:
     # Where Django looks for uploaded files
     oh_media_root: "/media"
     # Relative paths within oh_media_root, used by Django
+    oh_masters: "oh_masters"
```

Comment:
I expect to see this added to the generic `values.yaml` as well, for consistency.  It may not strictly be necessary but it's what we do for other specific values.